tic-tac-toe/src/Board.jsx

```diff
@@ -0,0 +1,70 @@
+import React, {useContext} from 'react';
+import Square from './Square';
+import BoardState from './contexts/BoardContext';
+import declareWinner from './contexts/declareWinner';
+
+const Board = (props) => {
+  const { turn, setTurn, boardState, setBoardState, history, setHistory} = useContext(BoardState);
+
+  const winner = declareWinner(boardState);
+  let status;
+  if (winner) {
+    status = `The winner is ${winner}!`;
+  }
+
+  function handleClick(num) {
```

Comment:
I think some of this logic might make more sense to live on the `context` instead of on the component. That way, the context would define not just the basic state but also an interface for updating the state. Totally up to you though - this is also a valid implementation, just a little bit different!